tsrc/cli/review.py

```diff
@@ -102,10 +127,20 @@ def log(self, job_id=None):
         logs = self.gl_helper.get_job_trace(self.project_id, job_id)
         print(logs)
 
+    def list_reviews(self):
+        mrs = sort_merge_requests(
+            self.gl_helper.get_project_merge_requests(self.project_id))
+        print(mrs)
```

Comment:
debug print? If not, use ui.info() instead

Reply:
done